docs/reference/index.rst

```diff
@@ -0,0 +1,17 @@
+API Reference
+================
+
+For as simple as the tutorials make ``ppb`` look there's a lot of power under
+the hood. This section will cover the raw what of the ppb API. To find out why
```

Comment:
> the raw what of the ppb API.

That's odd/confusing phrasing, IMO.
Not a blocker though.

Reply:
All of these literally exist as stubs, so yeah, better phrasing all around eventually.